samples/peripheral/802154_phy_test/src/rf_proc.c

```diff
@@ -283,8 +283,15 @@ void nrf_802154_cca_failed(nrf_802154_cca_error_t error)
 	k_work_submit(&rf_cca_failed_wq);
 }
 
+#if (NRF_802154_ENERGY_DETECTED_VERSION != 0)
+void nrf_802154_energy_detected(const nrf_802154_energy_detected_t *p_result)
+#else
```

Comment:
Use final api version only, in sdk-zephyr there ia also an unconditional migration
https://github.com/nrfconnect/sdk-zephyr/pull/1238/commits/eb7b5dceff859cf282779eaf6a0fb4d9633bd2b3

Reply:
fixed